homeassistant/components/mobile_app/logbook.py

```diff
@@ -0,0 +1,59 @@
+"""Describe mobile_app logbook events."""
+from __future__ import annotations
+
+from collections.abc import Callable
+
+from homeassistant.const import ATTR_FRIENDLY_NAME, ATTR_ICON
+from homeassistant.core import Event, HomeAssistant, callback
+
+from .const import DOMAIN
+
+IOS_EVENT_FINISHED_LAUNCHING = "ios.finished_launching"
+IOS_EVENT_ENTERED_BACKGROUND = "ios.entered_background"
+IOS_EVENT_BECAME_ACTIVE = "ios.became_active"
+IOS_EVENT_NOTIFICATION_ACTION_FIRED = "ios.notification_action_fired"
+IOS_EVENT_ACTION_FIRED = "ios.action_fired"
+IOS_EVENT_ZONE_ENTERED = "ios.zone_entered"
+IOS_EVENT_ZONE_EXITED = "ios.zone_exited"
```

Comment:
Isn't it weird that these are events? One could also listen to the person entity state changes.

CC @zacwest

Reply:
https://github.com/home-assistant/iOS/pull/926 documents the addition. tl;dr: not all zones are device_tracker zones, and sometimes automations need to know about region monitoring changes.